auth/token_verifier.go

```diff
@@ -100,9 +100,10 @@ type tokenVerifier struct {
 	expiredTokenCode  string
 	keySource         keySource
 	clock             internal.Clock
+	isEmulator        bool
```

Comment:
This might be too simple. It's better to implement custom `keySource` for emulator mode. 

But I don't sure custom `keySource` is necessary because
- We didn't decide how to verify tokens in the emulator mode yet.
- Cannot skip verification runs before signature verification.